Diagnostics/AVTester/Test-ExchAVExclusions.ps1

```diff
@@ -350,126 +444,44 @@ foreach ($extension in $extensionsList) {
 #Delete Random Folder
 Remove-Item $randomFolder
 
+$OutputPath = Join-Path $env:LOCALAPPDATA BadExclusions.txt
```

Comment:
Based off the issue, this should be using `$PSScriptRoot` instead of `$env:LOCALAPPDATA`

Reply:
make sure you also change the documentation as well.

Reply:
I found something weird when I was modifying this code. 
The code that write in LOCALAPPDATA is the function Write-SimpleLogFile but We have 2 functions with the same name and the same code one is under Diagnostics\AVtester and another one is on Performance\ExPerfWiz.
I will modify the one on EVTester to change write on LOCALAPPDATA and move to the path received.